djstripe/models.py

```diff
@@ -19,15 +19,16 @@
 import stripe
 
 from . import settings as djstripe_settings
-from .exceptions import SubscriptionCancellationFailure
+from .exceptions import SubscriptionCancellationFailure, SubscriptionApiError
 from .managers import CustomerManager, ChargeManager, TransferManager
 from .signals import WEBHOOK_SIGNALS
 from .signals import subscription_made, cancelled, card_changed
 from .signals import webhook_processing_error
+from .settings import ALLOW_MULTIPLE_SUBSCRIPTIONS, RETAIN_CANCELED_SUBSCRIPTIONS
```

Comment:
instead of directly importing, use djstripe_settings.ALLOW_MULTIPLE_SUBSCRIPTIONS and djstripe_settings.RETAIN_CANCELED_SUBSCRIPTIONS